cilium-dbg/cmd/encrypt_status.go

```diff
@@ -92,6 +93,9 @@ func dumpIPsecStatus() (models.EncryptionStatus, error) {
 	if err != nil {
 		return models.EncryptionStatus{}, fmt.Errorf("error getting IPsec decryption interfaces: %w", err)
 	}
+	if option.Config.TunnelingEnabled() && len(decryptInts) == 0 {
+		decryptInts = append(decryptInts, "cilium_vxlan")
```

Comment:
This conditional checks `TunnelingEnabled` but then prints the string `cilium_vxlan`. Not all tunneling modes are vxlan. The device should correspond to the tunnel device type in use.